src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/NocClaimantRepresentativeService.java

```diff
@@ -31,6 +39,11 @@ public class NocClaimantRepresentativeService {
     private final OrganisationClient organisationClient;
     private final AdminUserService adminUserService;
     private final NocCcdService nocCcdService;
+    private final NocNotificationService nocNotificationService;
+    private final CcdCaseAssignment ccdCaseAssignment;
+    private final CcdClient ccdClient;
+    private final NocHelper noCHelper;
```

Comment:
Just to keep the formatting consistent but think this will be moved anyway
```suggestion
    private final NocHelper nocHelper;
```